metrics/resizer.go

```diff
@@ -18,19 +18,19 @@ func init() {
 }
 
 type resizerSuccessResizeTotalAdapter struct {
-	metric prometheus.Counter
+	metric prometheus.CounterVec
 }
 
-func (a *resizerSuccessResizeTotalAdapter) Increment() {
-	a.metric.Inc()
+func (a *resizerSuccessResizeTotalAdapter) Increment(pvcname string, pvcns string) {
+	a.metric.With(prometheus.Labels{"persistentvolumeclaim": pvcname, "persistentvolumeclaimnamespace": pvcns}).Inc()
```

Comment:
We are using kube-state-metrics internally. It exposes `kube_persistentvolumeclaim_info` metric that uses the `namespace` label to describe where the PVC exists. I think it is better to use it because it is easy to join by promql.
https://github.com/kubernetes/kube-state-metrics/blob/master/docs/persistentvolumeclaim-metrics.md

Reply:
So just to confirm, you would rather see the label be namespace rather than persistentvolumeclaimnamespace? I'm perfectly fine with that but didn't know if that would be an issue due to namespace already being used today:

ex:
```
pvcautoresizer_success_resize_total{container="pvc-autoresizer", endpoint="metrics", instance="1.2.3.4:8080", job="pvc-autoresizer/pvc-autoresizer", namespace="pvc-autoresizer", pod="pvc-autoresizer-controller-84c96b8f9d-85pqd"}
```

If you're fine with overwriting that current value, I can get this updated pretty quickly.

Reply:
I think it is fine. The namespace label is assigned by prometheus, and we can manage which one is used by `honor_labels` prometheus configuration.